JavaDirectory/src/main/java/com/bug/free/invention/api/classes/Job.java

```diff
@@ -0,0 +1,109 @@
+package com.bug.free.invention.api.classes;
+
+public class Job {
+    int jobID;
+    String jobTitle;
+    String jobSpec;
+    int bandId;
+    int capabilityId;
+    String capabilityName;
+    String bandName;
+    String familyName;
+    int bandLevel;
```

Comment:
These should be `private`
```suggestion
    private int jobID;
    private String jobTitle;
    private String jobSpec;
    private int bandId;
    private int capabilityId;
    private String capabilityName;
    private String bandName;
    private String familyName;
    private int bandLevel;
```